.github/workflows/release.yml

```
@@ -19,22 +19,22 @@ jobs:
 
   release:
     runs-on: ubuntu-20.04
-    container:
-      image: xaynetci/yellow:9135de0
+    #container:
+      #image: xaynetci/yellow:9135de0
     timeout-minutes: 10
     needs: [build-ios-libs, build-android-libs]
     steps:
-      - name: Install SSH key
+      - name: Install SSH key gitlab
         uses: shimataro/ssh-key-action@3c9b0fc6f2d223b8450b02a0445f526350fc73e0 # v2.3.1
         with:
-          key: ${{ secrets.SSH }}
+          key: ${{ secrets.RELEASE_REPO_SSH_KEY }}
           known_hosts: ${{ secrets.KNOWN_HOSTS }}
 
       - name: Checkout repository
         uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
 
       - name: Setup job
-        uses: ./.github/actions/setup-job-docker
+        uses: ./.github/actions/setup-job
         with:
           dart: true
           rust: true
```

beacuse we use the old script. but maybe it doesn't matter because we probably won't get into that limit anymore
```suggestion
          rust: true
          github_token: ${{ secrets.GITHUB_TOKEN }}
```